Dockerfile

```diff
@@ -24,5 +25,9 @@ RUN wget https://releases.hashicorp.com/terraform/1.3.7/terraform_1.3.7_linux_am
 # Install task
 RUN curl -sSLf "https://github.com/go-task/task/releases/download/v3.21.0/task_linux_amd64.tar.gz" | tar -xz -C /usr/local/bin
 
+## uncomment for use as a local client
```

Comment:
Can this comment be expanded a bit? not sure what it means offhand when reading it -- also the following block is already not commented?

Reply:
I use the same image for the testing machines and for local connection to the testing machines as it is more convenient than clearing out my macbook every time. I forgot to stash the commenting out this go round

Reply:
Fixed!